data/json/furniture.json

```diff
@@ -3774,6 +3774,57 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_55gal_firebarrel",
+    "name": "fire barrel (200L)",
+    "symbol": "#",
+    "color": "red",
+    "looks_like": "55gal_drum",
+    "move_cost_mod": 2,
+    "coverage": 35,
+    "required_str": 8,
+    "flags": [ "PLACE_ITEM", "TRANSPARENT", "FIRE_CONTAINER", "EASY_DECONSTRUCT" ],
+    "deployed_item": "55gal_firebarrel",
+    "examine_action": "fireplace",
+    "max_volume": 4000,
+    "bash": {
+      "str_min": 8,
+      "str_max": 30,
+      "sound": "metal screeching!",
+      "sound_fail": "clang!",
+      "items": [
+        { "item": "scrap", "count": [ 8, 20 ] },
+        { "item": "sheet_metal_small", "count": [ 3, 10 ] },
+        { "item": "sheet_metal", "count": [ 0, 1 ] }
+      ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_30gal_firebarrel",
+    "name": "fire barrel (100L)",
+    "symbol": "#",
+    "color": "red",
+    "looks_like": "30gal_drum",
+    "move_cost_mod": 2,
+    "coverage": 35,
+    "required_str": 8,
+    "flags": [ "PLACE_ITEM", "TRANSPARENT", "FIRE_CONTAINER", "EASY_DECONSTRUCT" ],
+    "deployed_item": "30gal_firebarrel",
+    "examine_action": "fireplace",
+    "max_volume": 4000,
+    "bash": {
+      "str_min": 8,
+      "str_max": 30,
+      "sound": "metal screeching!",
+      "sound_fail": "clang!",
+      "items": [
```

Comment:
Should be one line:
`"items": [ { "item": "scrap", "count": [ 5, 15 ] }, { "item": "sheet_metal_small", "count": [ 1, 9 ] } ]`

Reply:
Alright I'll do that. But all the other recipes have it this way. I really just copy pasted the other recipe. Wouldn't it be a problem that my recipes are formatted differently?

Reply:
That's the last change that JSON-formatter wants.